core/renderers/zelos/info.js

```diff
@@ -189,30 +191,30 @@ RenderInfo.prototype.getInRowSpacing_ = function(prev, next) {
 RenderInfo.prototype.getSpacerRowHeight_ = function(
     prev, next) {
   // If we have an empty block add a spacer to increase the height.
-  if (Blockly.blockRendering.Types.isTopRow(prev) &&
-      Blockly.blockRendering.Types.isBottomRow(next)) {
+  if (Types.isTopRow(prev) &&
+      Types.isBottomRow(next)) {
     return this.constants_.EMPTY_BLOCK_SPACER_HEIGHT;
   }
   const followsStatement =
-      Blockly.blockRendering.Types.isInputRow(prev) && prev.hasStatement;
+      Types.isInputRow(prev) && prev.hasStatement;
   const precedesStatement =
-      Blockly.blockRendering.Types.isInputRow(next) && next.hasStatement;
+      Types.isInputRow(next) && next.hasStatement;
   if (precedesStatement || followsStatement) {
     const cornerHeight = this.constants_.INSIDE_CORNERS.rightHeight || 0;
     const height = Math.max(this.constants_.NOTCH_HEIGHT, cornerHeight);
     return precedesStatement && followsStatement ?
         Math.max(height, this.constants_.DUMMY_INPUT_MIN_HEIGHT) : height;
   }
   // Top and bottom rows act as a spacer so we don't need any extra padding.
-  if ((Blockly.blockRendering.Types.isTopRow(prev))) {
+  if ((Types.isTopRow(prev))) {
```

Comment:
Extra parenthesis? (Looks like this is from before, but might be good to fix.)

Reply:
I can remove these and the others.